### ISSUE TITLE
[Bug]: Upon accepting an e-mail (ICS) calendar invite from Gmail through Thunderbird, the calendar entry is added but the RSVP e-mail is never sent

### ISSUE THREAD

**Issue by Rudd-O** [nextcloud/server#38055]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When I open an e-mail with an invite from a Gmail user (not tested with other calendaring systems), the top blue bar of the message gives me options to accept, tentative or decline (pretty much standard).

If I hit accept, or tentative, nothing is sent back to the user, although the entry is in fact created, with the correct status on the Nextcloud Web calendar (it shows as accepted or tentative based on what I selected on Thunderbird).  As a consequence of the acceptance / tentative email never being sent, people inviting me to events never get an update and therefore never know if I am attending or not.

For Nextcloud to actually send a notification, I must then open the calendar entry on Thunderbird and change its status from tentative to accepted, or from accepted to tentative -- then, and only then, the Nextcloud server will deign to send e-mail with the updated status.

One final issue which is almost certainly related: if I make the change from accepted to tentative, or from tentative to accepted, through the Nextcloud Web calendar interface, no notification is ever sent to the organizer.

(Client-side scheduling is turned off in Thunderbird.)

### Steps to reproduce

Colleague makes calendar entry with me as invitee.

I get email with ICS from his software (Gmail).

I use Thunderbird to _accept_ and thereby register the calendar entry to my Nextcloud calendar.

### Expected behavior

I expect Nextcloud to send invitation acceptances as soon as my e-mail client files the event received through e-mail into my calendar (and I am obviously NOT the organizer of the event).

Nextcloud clearly has the ability to do this, since ex-post-facto changing the RSVP status of the event (on Thunderbird) triggers the sending of an e-mail _coming specifically from Nextcloud_ rather than from Thunderbird.

### Installation method

Other Community project

### Nextcloud Server version

25.0.3

### Operating system

RHEL/CentOS

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Fresh Nextcloud Server install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
sudo -u apache php occ config:list system
The current PHP memory limit is below the recommended value of 512MB.
{
    "system": {
        "log_type": "syslog",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "updatechecker": false,
        "check_for_working_htaccess": false,
        "asset-pipeline.enabled": false,
        "assetdirectory": "\/var\/lib\/nextcloud",
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "apps_paths": [
            {
                "path": "\/usr\/share\/nextcloud\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/lib\/nextcloud\/apps",
                "url": "\/apps-appstore",
                "writable": true
            }
        ],
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.dragonfear"
        ],
        "dbtype": "mysql",
        "version": "25.0.3.2",
        "overwrite.cli.url": "http:\/\/cloud.dragonfear",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "htaccess.RewriteBase": "\/",
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "tls",
        "mail_sendmailmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpauth": 1,
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***"
    }
}
```

### List of activated Apps

```shell
sudo -u apache php occ app:list
The current PHP memory limit is below the recommended value of 512MB.
Enabled:
  - activity: 2.17.0
  - calendar: 4.3.3
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contacts: 5.2.0
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - news: 21.1.0
  - nextcloud_announcements: 1.14.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - photos: 2.0.1
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflowengine: 2.7.0
Disabled:
  - admin_audit
  - bruteforcesettings
  - encryption
  - files_external
  - suspicious_login
  - twofactor_totp
  - user_ldap
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

There's nothing relevant in the logs about this.  I do however confirm that e-mail is not being sent when I was expecting it to be sent, and is being sent in the other cases, because I can see my mail server logs and I can see / fail to see, the e-mails coming into the other person's inbox.

**Comment by Rudd-O** [comment]
https://github.com/nextcloud/server/issues/33188 describes what happens to me, except I tested that changing the attendance from Thunderbird — once the event has entered the calendar — does in fact send notifications (and doing the same through the Nextcloud Web calendar does not).

**Comment by Rudd-O** [comment]
I've also confirmed that accepting the invite through Etar (which is backed by DavX5) also never sends calendar attendance information back to the sender through Nextcloud.  Same goes with Simple Calendar by Tibor Kaputa.

**Comment by Rudd-O** [comment]
Effectively, the feature should be that Nextcloud does the scheduling when events are added and RSVP status is changed.  That way, any invitations received can be added with any mail client to any calendaring suite, and the mail client does not need to send scheduling-related e-mails.

Note that this would probably require the user to input his e-mail server and credentials in Nextcloud user settings, so that Nextcloud can send e-mail on his behalf.

**Comment by flortsch** [comment]
Same issue is reported here as well: https://github.com/nextcloud/server/issues/38651

**Comment by Rudd-O** [comment]
> #38651 

Slightly different issue but related.

**Comment by SebastianKrupinski** [comment]
Fixed with #44893
